pact3.0/03-coin-interface.repl

```diff
@@ -0,0 +1,83 @@
+; --------------------------------------------------------------------------
+; --------------------------------------------------------------------------
+; Using formal verification in Pact interfaces to create sample coin contract.
+
+
+; --------------------------------------------------------------------------
+; Installs account schema module
+; --------------------------------------------------------------------------
+(begin-tx)
+(env-data { "admin-keyset": ["product-dept" "dev-dept"] })
+
+(module acct-module ACCT-GOVERNANCE
+  @doc "account schema module"
+
+  (defcap ACCT-GOVERNANCE ()
+    (enforce-guard (read-keyset "admin-keyset")))
+
+  (defschema account
+    balance:integer
+    ks:keyset
+    active:bool
+  )
+)
+(commit-tx)
+
+; --------------------------------------------------------------------------
+; Installs coin abstract interface
+; --------------------------------------------------------------------------
+(begin-tx)
+(interface coin-sig
+
+  "Coin Contract Abstract Interface Example"
+
+  (use acct-module)
+
+  (defun transfer:string (from:string to:string amount:integer)
+    @doc   "Transfer money between accounts"
+    @model [(property (row-enforced accounts "ks" from))
+            (property (> amount 0))
+            (property (= 0 (column-delta accounts "balance")))
+            ]
+  )
+)
+(commit-tx)
+
+; --------------------------------------------------------------------------
+; Installs hello-coin module
+; --------------------------------------------------------------------------
+(begin-tx)
+
+(module hello-coin HELLO-GOVERNANCE
+  @doc "Implementaion of hello-coin based on coin-sig interface"
+
+  (use acct-module)
+  (implements coin-sig)
+
+  (defcap HELLO-GOVERNANCE ()
+    (compose-capability (ACCT-GOVERNANCE)))
```

Comment:
Again, make a "normal function" `(enforce-acct-admin)` in `acct-module` and call that instead, since we'll be fixing the bug that allows this to run

Reply:
Or just do `true` here